tests/e2e/pom/paper-view-model.ts

```diff
@@ -0,0 +1,34 @@
+import { expect, type Locator, type Page } from "@playwright/test";
+
+export class DevPaperViewPage {
+    readonly page: Page;
+    readonly allListEntries: Locator;
+    readonly referenceListEntry0: Locator;
+
+    constructor(page: Page) {
+        this.page = page;
+        this.allListEntries = page.getByRole("listitem");
+        this.referenceListEntry0 = page
+            .getByRole("button", {
+                name: /Paper 0 to be referenced/,
```

Comment:
You can actually just use a string instead of the regex, since you do not use any regex parts.

```suggestion
                name: "Paper 0 to be referenced",
```

If you had used `name: "Paper 0 to be referenced", exact: true`, then this would make a difference, but without it, it does not need to be exact.

Reply:
Yes. That is just a remnant of previous tries.